app/Console/Commands/CreateProjectSnapshot.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Project;
+use App\Projects;
+use App\Snapshot;
+use Illuminate\Console\Command;
+
+class CreateProjectSnapshot extends Command
+{
+    protected $signature = 'snapshot:today {--f|force : Snapshot all projects, updating any existing data for today}';
+
+    protected $description = 'Create a snapshot of the current status of all GitHub projects.';
+
+    protected $bar;
+
+    public function handle()
+    {
+        // Get a collection of projects from projects.json that don't have a snapshot for today
+        $projects = $this->getProjectsToSnapshot();
```

Comment:
```suggestion
        $projects = $this->getProjectsToSnapshot();
```

No need for that comment